### DISCUSSION TITLE
back-to-basics-mmp-mvp-poc

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#25] [Technical Blog]
# back-to-basics-mmp-mvp-poc

https://wsbctechnicalblog.github.io/back-to-basics-mmp-mvp-poc.html

**Comment by jaskaran27177** [comment]
Nice job clarifying MMP, MVP, and POC! Your article was clear and examples make it so much easier to understand their unique roles in product development. 
Question: How can teams effectively balance the use of TLAs to maintain clarity without overburdening communication?

**Reply by wpschaub** [reply]
@jaskaran27177 I hope I understand your question correctly; if not, please let me know.

In our group, we developed a common language glossary that includes frequently used terms in both business and engineering, as well as acronyms and their definitions. It is fascinating how conversations and expectations can diverge due to misunderstood acronyms or common terms like "project." For example, "Azure DevOps project," "Visual Studio project," and "business project" likely mean different things.

We have also cultivated a culture within our team that encourages questioning all TLAs (two/three-letter acronyms). We make it a point to spell out acronyms the first time they are used and avoid using acronyms when speaking. For instance, when reading aloud "the PoC will be fun," we actually say "the proof-of-concept will be fun." 

In DevOps, as in other areas, PEOPLE are the main (80% people, 15% process, 5% products) focus. You must skillfully and constantly foster the practice of communication and the usage (or not) of acronyms.
